packages/fx-core/tests/common/local/localSettingsHelper.test.ts

```diff
@@ -25,7 +25,7 @@ describe("localSettingsHelper", () => {
         hostType: "Azure",
         azureResources: [] as string[],
         capabilities: ["Tab"],
-        activeResourcePlugins: ["fx-resource-aad-app-for-teams", "fx-resource-simple-auth"],
+        activeResourcePlugins: ["fx-resource-aad-app-for-teams"],
```

Comment:
did we remove simple auth plugin from all test cases? Since we still support simple auth enabled project, we'd better keep some test cases for simple auth enabled scenarios to ensure changes won't break simple auth scenario

Reply:
Added several tests. Thanks!